projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.html

```diff
@@ -1,19 +1,44 @@
-<span
-  *ngFor="let item of items"
-  class="loader"
-  [attr.aria-label]="ariaLabel"
-  aria-busy="true"
-  aria-valuemin="0"
-  aria-valuemax="100"
-  [attr.aria-valuetext]="loadingText"
-  role="progressbar"
-  tabindex="0"
-  [ngClass]="{
+<div *ngIf="appearance  !== 'customContent'" >
+  <span
+    *ngFor="let item of items"
+    class="loader"
+    [attr.aria-label]="ariaLabel"
+    aria-busy="true"
+    aria-valuemin="0"
+    aria-valuemax="100"
+    [attr.aria-valuetext]="loadingText"
+    role="progressbar"
+    tabindex="0"
+    [ngClass]="{
     circle: appearance === 'circle',
-    progress: animation === 'progress',
-    'progress-dark': animation === 'progress-dark',
-    pulse: animation === 'pulse'
-  }"
-  [ngStyle]="theme"
->
-</span>
+      progress: animation === 'progress',
+      'progress-dark': animation === 'progress-dark',
+      pulse: animation === 'pulse'
+    }"
+    [ngStyle]="theme"
+  >
+  </span>
+</div>
+
+<div  *ngIf="appearance  === 'customContent'" >
+  <div
+    class="loader customContent"
+    [attr.aria-label]="ariaLabel"
+    aria-busy="true"
+    aria-valuemin="0"
+    aria-valuemax="100"
+    [attr.aria-valuetext]="loadingText"
+    role="progressbar"
+    tabindex="0"
+    [ngClass]="{
```

Comment:
Could explain few things here, please? When using a customContent these classes shouldn't even exist, correct? So that, we can make sure  we won't have misconfiguration in that matter. Could you remove it, please? If this is true, it probably won't need `ngStyle` as well.

Perhaps, we could combine these 2 elements into a single one and make this markup cleaner? If we have a class added dinamically when `appearance === 'customContent'` we can get rid of double elements, having everything related to the `div` specific styling added and managed as a class. So that, we can have dinamic content being switch, if necessary.

Could you check that, please?